docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -148,6 +148,7 @@ There you can find links to upgrade notes for other versions too.
             </copy>
         </target>
         ```
+    - download [`RedisVersionsFacadeTest.php`](https://github.com/shopsys/project-base/tree/master/tests/ShopBundle/Functional/Component/Redis/RedisVersionsFacadeTest.php) to your tests directory `tests/ShopBundle/Functional/Component/Redis/`
```

Comment:
consider the use of master branch....

Reply:
it's part of release process to change it. Right now I don't have any different target to link to